### DISCUSSION TITLE
Help bypassing default theme options

### DISCUSSION THREAD

**Discussion by theroboteenager** [nunocoracao/blowfish#21] [Q&A] [answered]
Hello so I want to create a new color theme and not use the theme that are already available.

I have create the custom.css file but what do i need to do in order the color change.

Is there a setting that i need to  modify in any of the .toml files?

Any help would be highly appreciated.

**Comment by nunocoracao** [comment, marked as answer]
Hello @theroboteenager, depending on how much you want to change (and how) it might just be a case of adding a new file to a folder or having to re-build the entire CSS. This section of the docs explains exactly what you need to do in each of those situations. Let me know if you have further questions
https://nunocoracao.github.io/blowfish/docs/advanced-customisation/
